ckcp/openshift_dev_setup.sh

```diff
@@ -363,13 +363,16 @@ install_pipeline_service() {
 }
 
 register_compute() {
+  resources="$(printf '%s,' "${CRS_TO_SYNC[@]}")"
```

Comment:
I'm not familiar with the syntax of these 2 lines, can you please explain what they do?